projects/batfish/src/main/java/org/batfish/bdp/VirtualRouter.java

```diff
@@ -644,10 +644,12 @@ public void initStaticRib() {
         continue;
       }
       // interface route
-      if (sr.getNextHopIp().equals(Route.UNSET_ROUTE_NEXT_HOP_IP)) {
-        _staticInterfaceRib.mergeRoute(sr);
-      } else {
-        _staticRib.mergeRoute(sr);
+      if (sr.getNextHopIp() != null) {
```

Comment:
per our discussion:

* please add unit test
* please add `@Nullable` annotation to the getter `sr.getNextHopIp()` so that we know it can be null.